### PR TITLE
Update README with build info

### PR DESCRIPTION
**Pull request by emperorcezar**
**Issue Link**

https://spothero.atlassian.net/browse/MONOLITH-140

**Description**

ROI on building up a concourse job for this isn't there yet. Just documenting how to build and push it.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,6 +34,16 @@ command::
...
+  
+  
+Building
+==========
+Build the container with
+`docker build -f Dockerfile-celery3 -t spothero/redis-celery-exporter:$VERSION --build-arg BUILD_VERSION=$VERSION .`
+replacing `$VERSION` with the version you're building
+
+Push container with
+`docker push spothero/redis-celery-exporter:$VERSION`
```
Comment (theisenmark) [inline README.rst]: Should this come before/after the "How to use" section? Or is it meant to be in the middle?

**Review by theisenmark — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -34,6 +34,16 @@ command::
 
   $ celery-prometheus-exporter
   Starting HTTPD on 0.0.0.0:8888
+  
+  
+Building
+==========
+Build the container with
+`docker build -f Dockerfile-celery3 -t spothero/redis-celery-exporter:$VERSION --build-arg BUILD_VERSION=$VERSION .`
+replacing `$VERSION` with the version you're building
```
Comment (ShadyPenguin) [inline README.rst]: What is the current version? Can you provide an example of what VERSION might be?

File: README.rst (hunk 1 above)
Comment (emperorcezar) [reply README.rst]: Fixed this, don't know how it got there like that.

File: README.rst (hunk 2 above)
Comment (emperorcezar) [reply README.rst]: I clarified a bit and formatted. This is a public fork so I don't know to go into too much detail of our internals.

**Pull request merged**